apps/src/componentLibrary/typography/typography.module.scss

```diff
@@ -32,7 +36,37 @@
 // identical classes for potential use on other elements if needed.
 
 // ----------------------------------------
-// v 1.0.1
+// v 2.0.0
+// ----------------------------------------
+
+// Sizes
+// Most of the typography elements has a set of different sizes (defined by "-one", "-two", "-three", "-four" prefixes).
```

Comment:
super nits:
- `have a set` 
- `suffixes` rather than `prefixes`

Reply:
Super nice catch, thank you! I was thinking about suffixes all the time while implementing this, I wonder how I haven't spotted that 😄